content/zh-cn/docs/concepts/containers/images.md

```diff
@@ -110,12 +110,8 @@ these values have:
 : the image is pulled only if it is not already present locally.
 
 `Always`
-: every time the kubelet launches a container, the kubelet queries the container
-  image registry to resolve the name to an image
-  [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). If the kubelet has a
-  container image with that exact digest cached locally, the kubelet uses its cached
-  image; otherwise, the kubelet pulls the image with the resolved digest,
-  and uses that image to launch the container.
+: every time the kubelet launches a container, the kubelet queries the container image registry to resolve the name to an image
+  [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). The kubelet then pulls the image with the resolved digest, and uses that image to launch the container.
```

Comment:
Please don't update the zh-cn localization. The commented text needs a translation and the localization team has its own pace on this.